nsbno/spor
# Give docs a facelift

## Background

Some bugs on the layout for the docs needed some attention.

## Solution

- Layout is centered on responsive and big screens
- Resolved layout bleeding
- Resolved code bleeding
- Moved search to top bar and added a keyboard shortcut (layout inspired by react.dev)
- Made all sidebars fixed on page
- Header is fixed
- Other small warnings for layout

## UU checks


Note: To trigger pipeline for the documentation site (spor.vy.no) you need to bump the version.

## How to test

Pull repo and tap around on page. The Toast page was very hard to fix, should be tested by more people.

## Screenshots

| Before | After |
| ------ | ----- |
| ![image](https://github.com/user-attachments/assets/298c5cf8-509a-49de-be65-33fbfa35ae0c) | ![image](https://github.com/user-attachments/assets/1c93fdc0-639f-4cfa-b7b5-ded4e8f9a9d2) |

### Small demo:

![spor-demo-layout](https://github.com/user-attachments/assets/4fe17153-c851-4d01-9473-c7aba35b242d)
